twittermap/app/views/twittermap/main.scala.html

```diff
@@ -24,6 +24,14 @@
       wsURL:"@ws",
       sentimentEnabled: @sentimentEnabled,
       sentimentUDF: "@sentimentUDF",
+      startDate: new Date("@startDate"),
+      @if(isDrugMap) {
+        mapLegend: "$",
+        sumText : "dollars",
+      } else {
+        mapLegend: "Count:",
```

Comment:
I think the sentiment related logic has covered this case. I don't want to make this half scala half html file too complicated.